dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -50,6 +47,9 @@ postParamSaveMethod.desc=Specifies where POST parameters are stored upon redirec
 postParamCookieSize=POST parameter cookie size
 postParamCookieSize.desc=Size of the POST parameter cookie. If the size of the cookie is larger than the browser limit, unexpected behavior may occur. The value of this property must be a positive integer and represents the maximum size of the cookie in bytes. The equivalent custom property in the full application server profile is com.ibm.websphere.security.util.postParamMaxCookieSize.
 
+postParamMaxRequestBodySize=Maximum request body size to save POST parameters
+postParamMaxRequestBodySize.desc=Specifies the maximum size of the request body of an HTTP POST request to save request parameters. If the content length of the request is larger than the maximum allowed size, any POST parameters in the request will not be saved. If the content length of the request is unknown, request parameters will not be saved regardless of the value of this attribute. Setting this attribute to -1 means that POST parameters will always be saved as long as the content length of the request is known.
```

Comment:
- Change "will not be" to "are not"
- Replace "will ... be" and "as long as" (presumably, the "long" doesn't refer to length) in the last sentence, for example: "Setting this attribute to -1 means that POST parameters are always saved if the content length of the request is known."

Reply:
Missed that one, sorry.